modules/setting/indexer.go

```diff
@@ -40,6 +41,7 @@ var Indexer = struct {
 	IssueIndexerName: "gitea_issues",
 
 	RepoIndexerEnabled: false,
+	RepoIndexerUnits:   []string{"sources", "forks", "mirrors", "templates"},
```

Comment:
The default value is only `sources` according `app.example.ini`

Reply:
`app.example.ini` has all of them https://github.com/go-gitea/gitea/pull/23187/files#diff-eda333ab2034231553808ef0f99e28cd1f209c7f583fd3d0aa0fb1b77e573cd4R1386